packages/gatsby-source-nacelle/gatsby-node.js

```diff
@@ -0,0 +1,212 @@
+const fs = require('fs-extra');
+const { print } = require('gatsby/graphql');
+const { createRemoteFileNode } = require('gatsby-source-filesystem')
+const {
+  sourceAllNodes,
+  createSchemaCustomization,
+  compileNodeQueries,
+  readOrGenerateDefaultFragments,
+  buildNodeDefinitions,
+  loadSchema,
+  createDefaultQueryExecutor
+} = require('gatsby-graphql-source-toolkit');
+require('dotenv').config();
+
+const CHUNK_SIZE = 100;
+const fragmentsDir = process.cwd() + `/nacelle-fragments`;
+
+const PaginateNacelle = {
+  name: 'NacellePagination',
+  expectedVariableNames: ['first', 'after'],
+  start() {
+    return {
+      variables: { first: CHUNK_SIZE, after: undefined },
+      hasNextPage: true
+    };
+  },
+  next(state, page) {
+    return {
+      variables: { first: CHUNK_SIZE, after: page.nextToken },
+      hasNextPage: Boolean(page.nextToken && page.items.length === CHUNK_SIZE)
+    };
+  },
+  concat(acc, page) {
+    return {
+      ...acc,
+      items: {
+        ...acc.items,
+        ...page.items
+      },
+      nextToken: page.nextToken
+    };
+  },
+  getItems(pageOrResult) {
+    return pageOrResult.items;
+  }
+};
+
+async function createSourcingConfig(gatsbyApi, pluginOptions) {
+  const { verbose } = pluginOptions
```

Comment:
Missing semi-colon -- Using prettier?

Reply:
Yes, but local settings were interfering. I'll update the project to add some VSCode config to make sure that the project settings override any others.

Reply:
Kk. Do you mind gitignoring those after you add them? I had the opposite problem in the other example repo where the local settings were messing up prettier. I think it's safest just to let people fix those on their own if it's an issue. Thoughts?

Reply:
> I had the opposite problem in the other example repo where the local settings were messing up prettier

😭 Some day, they're gonna figure out how to make it all work. In the meantime, sure, I'll keep it local and will make sure the config is kept out of source control.